go-controller/pkg/ovn/libovsdbops/chassis.go

```diff
@@ -0,0 +1,73 @@
+package libovsdbops
+
+import (
+	"context"
+	"fmt"
+
+	libovsdbclient "github.com/ovn-org/libovsdb/client"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/sbdb"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+)
+
+// findChassis returns all the logical chassis
+func ListChassis(sbClient libovsdbclient.Client) ([]sbdb.Chassis, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), types.OVSDBTimeout)
+	defer cancel()
+	searchedChassis := []sbdb.Chassis{}
+	err := sbClient.List(ctx, &searchedChassis)
+	if err != nil {
+		return nil, fmt.Errorf("failed listing chassis err: %v", err)
+	}
+
+	return searchedChassis, nil
+}
+
+func deleteChassis(sbClient libovsdbclient.Client, chassisNames ...string) error {
+	var opModels []OperationModel
+
+	for _, chassisName := range chassisNames {
+		opModels = append(opModels, OperationModel{
+			Model: &sbdb.Chassis{
+				Name: chassisName,
+			},
+		})
+	}
+
+	m := NewModelClient(sbClient)
+	if err := m.Delete(opModels...); err != nil {
+		return err
+	}
+	return nil
+}
+
+func DeleteStaleChassis(sbClient libovsdbclient.Client, chassisHostNameMap map[string]string) error {
+	staleChassisNames := []string{}
+	for _, v := range chassisHostNameMap {
+		staleChassisNames = append(staleChassisNames, v)
+	}
+
+	if err := deleteChassis(sbClient, staleChassisNames...); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
nit: I don't think this method fits the package.

Reply:
I need to update in response to Tim's Comments anyways, so I can change this around

Reply:
Done